week-1-practise/level-1/8-function-output-1.js

```diff
@@ -7,6 +7,10 @@ console.log(result);
 
 // Answer these questions:
 // - What does this program do?
+// It returns a rundom number in range 0-1(inclusive of 0, but not 1) and mulplies it by 10.
```

Comment:
It might be better to just say that the program prints out a random number in the range [0, 10) (i.e. between zero (inclusive) up to but not including 10).